server/bleep/migrations/20230831165918_templates.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE templates (
+    id TEXT NOT NULL,
+    name TEXT NOT NULL,
+    modified_at DATETIME NOT NULL DEFAULT (datetime('now')),
+    content TEXT NOT NULL
+);
+
+INSERT INTO templates (
+  id,
+  name,
+  content
+) VALUES (
+  'DEFAULT_UNIT_TEST',
+  'Write a test',
+  'Perform the following steps:\n\nStep 1: Describe the purpose of the code\nStep 2: Identify which functions would benefit from unit testing\nStep 3: Write a unit test for each of those functions'
```

Comment:
new lines get double escaped and return as \\n to the frontend which breaks rendering